benchmarking-a-deployment.md

```diff
@@ -0,0 +1,211 @@
+2019/01/21 11:15:25 Removing container bh2qi2fo3v1lt1ft4neg ...
+2019/01/21 11:15:25 Removing container bh2qi7fo3v1lt1ft4ngg ...
+2019/01/21 11:15:25 Removing container bh2qitvo3v1lt1ft4npg ...
+2019/01/21 11:15:25 Removing container bh2qievo3v1lt1ft4njg ...
+2019/01/21 11:15:25 Removing container bh2qj2vo3v1lt1ft4nrg ...
+2019/01/21 11:15:25 Removing container bh2qj0fo3v1lt1ft4nqg ...
+2019/01/21 11:15:25 Removing container bh2qicfo3v1lt1ft4nig ...
+2019/01/21 11:15:25 Removing container bh2qihfo3v1lt1ft4nkg ...
+2019/01/21 11:15:25 Removing container bh2qijvo3v1lt1ft4nlg ...
+2019/01/21 11:15:25 Removing container bh2qimfo3v1lt1ft4nmg ...
+2019/01/21 11:15:25 Removing container bh2qiovo3v1lt1ft4nng ...
+2019/01/21 11:15:25 Removing container bh2qi4vo3v1lt1ft4nfg ...
+2019/01/21 11:15:25 Removing container bh2qirfo3v1lt1ft4nog ...
+2019/01/21 11:15:25 Removing container bh2qhvvo3v1lt1ft4ndg ...
+2019/01/21 11:15:25 Removing container bh2qi9vo3v1lt1ft4nhg ...
+2019/01/21 11:15:39 Containers boot time measurement:
+2019/01/21 11:15:39   Launching all 15 containers took 2m36.560310342s
+2019/01/21 11:15:39   Out of 15 containers 0 failed to launch
+2019/01/21 11:15:39   Average container launch time: 6.149119411s
+2019/01/21 11:15:39   Max container launch time: 6.576302043s
+2019/01/21 11:15:39   Min container launch time: 5.911184959s
+2019/01/21 11:15:39   Android system failed to boot in the following containers:
+2019/01/21 11:15:39     None
+2019/01/21 11:15:39 Containers statistics:
+2019/01/21 11:15:39   FPS avg 58 min 52 max 64 for 15 containers
+2019/01/21 11:15:39   Containers below run at low FPS(<30):
+2019/01/21 11:15:39     None
+```
+
+The benchmark command launches 15 containers on the Anbox `null` platform with the following default display specification:
+
+ | Display Spec            | Value |
+ | ----------------------- | ----- |
+ | Display width           | 1280  |
+ | Display height          | 720   |
+ | FPS                     | 60    |
+ | Display density         | 160   |
+
+
+You can configure a different display specification through the `--user-data` parameter when running the benchmark. The required format for the parameter varies based on the different platforms:
+
+| Platform              | Required format of user data          | Example                                                   |
+| --------------------- | ------------------------------------  | --------------------------------------------------------  |
+| Null                  | Comma-separated values                | <display_width>,<display_height>,<display_fps>,<display_density>  |
+| Swrast                | Comma-separated values                | <display_width>,<display_height>,<display_fps>,<display_density>  |
+| WebRTC                | Json-based                            | {<br>"display_width": <display_width>,<br>"display_height": <display_height>,<br>"display_density": <display_density>,<br>"fps": <display_fps>,<br>"render_only": true<br> } |
```

Comment:
Let's remove those `<br>`, they are probably a result of some bad templating

Reply:
Per discussion in MP,  this is wrong.